generators/app/configs/package.json.js

```diff
@@ -33,11 +37,15 @@ module.exports = function(generator) {
       node: `^${major}.0.0`,
       [packager]: version
     },
-    'scripts': {
-      test: `${packager} run eslint && ${packager} run ${props.tester}`,
-      eslint: `eslint ${lib}/. test/. --config .eslintrc.json`,
+    scripts: {
+      test: `${packager} run lint && ${packager} run ${props.tester}`,
+      lint: lintScript,
```

Comment:
If we merge this, we're surely going to get more options in the future.  Could you change `lintScript` to an object and set `lint: lintScript[props.linter]`, here?  I'm sure the next request will be to add Prettier.  ;)

Reply:
Also, with both eslint and standard, I always set `npm run lint` with the `--fix` flag. It never broke my code, so I guess that's what people would always want, just like with prettier. I didn't change the eslint script here. I thought that could be a discussion for another issue, but maybe that should also change now:

`eslint --fix ${lib}/. test/. --config .eslintrc.json`

Reply:
I wouldn't want `--fix` running on my CI server, though.

Reply:
You can usually do `yarn lint --fix`, right?  That's how I normally do it locally.

Reply:
I guess I've never thought about the CI case. And I didn't know I could forward arguments to run-script, I just learned something new. Thanks for your review.